src/huggingface/forte/huggingface/token_classification.py

```diff
@@ -0,0 +1,221 @@
+# Copyright 2021 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Wrapper of the Token Classification models on HuggingFace platform (context
+understanding)
+"""
+import importlib
+import re
+from typing import Dict, Set
+from transformers import pipeline
+
+from forte.common import Resources
+from forte.common.configuration import Config
+from forte.data.data_pack import DataPack
+from forte.processors.base import PackProcessor
+from forte.common import ProcessorConfigError
+from forte.utils.tagging_scheme import bio_merge
+
+__all__ = [
+    "TokenClassification",
+]
+
+
+class TokenClassification(PackProcessor):
+    r"""Wrapper of the models on HuggingFace platform with pipeline tag of
+    `ner`.
+    https://huggingface.co/models?pipeline_tag=token-classification
+    This wrapper could take any model name on HuggingFace platform with pipeline
+    tag of `ner` in configs to make prediction on the user
+    specified entry type in the input pack and the prediction result goes to the
+    user specified attribute name of that entry type in the output pack. User
+    could provide task and models in the config to perform NER, POS,
+    word segmentation task, etc.
+
+    """
+
+    def __init__(self):
+        super().__init__()
+        self.classifier = None
+
+    def set_up(self):
+        self.classifier = pipeline(
+            "ner",
+            model=self.configs.model_name,
+            tokenizer=self.configs.tokenizer,
+            framework=self.configs.framework,
+            device=self.configs.cuda_device,
+        )
+
+    def initialize(self, resources: Resources, configs: Config):
+        super().initialize(resources, configs)
+        self.set_up()
+
+    def _process(self, input_pack: DataPack):
+        """Perform HuggingFace NER Pipeline on the input data pack.
+
+        Args:
+            input_pack: Input pack to fill
+        Returns:
+        """
+        input_path, input_module = self.configs.entry_type.rsplit(".", 1)
+        mod = importlib.import_module(input_path)
+        input_entry = getattr(mod, input_module)
+
+        output_path, output_module = self.configs.output_entry_type.rsplit(
+            ".", 1
+        )
+        mod = importlib.import_module(output_path)
+        output_entry = getattr(mod, output_module)
+
+        for entry_specified in input_pack.get(entry_type=input_entry):
```

Comment:
The `get` function supports to get entry from strings directly. You don't have to deal with the modules and classes. 
```suggestion
        for entry_specified in input_pack.get(entry_type= self.configs.entry_type):
```